_docs/concepts/what-is-istio/comparison.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Istio vs. Other Software
+overview: A comparison between Istio's and similar software.
+              
+order: 30
+
+layout: docs
+type: markdown
+---
+
+This page provides a comparison between Istio and similar software.
+
+## Istio vs. Envoy
+
+Envoy is one of the building blocks of Istio. In Istio, an Envoy instance is deployed as a sidecar to every running service. Together, this set of Envoy instances makes up the **data plane**.
+
+Istio provides, amongst other things, several components which enable the data plane to retrieve and locally apply global policies (routing, load balancing, service discovery, etc.). Those components (the two main ones being [Mixer]({{home}}/docs/concepts/policy-and-control/mixer.html) and [Pilot]({{home}}/docs/concepts/traffic-management/pilot.html)) implement the Envoy [LDS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/lds.proto), [CDS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/cds.proto), [EDS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/eds.proto), and [RDS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/rds.proto) APIs, and they form the **control plane** of the service mesh.
```

Comment:
I think this needs to be expanded, even though you link to the overview page from here. We should describe all the things Istio adds to Envoy.

Here's a rough outline:
- An extensible implementation of Envoy's Data Plane APIs (Pilot) supporting rich routing rules and resiliency controls
- An extensible implementation of a Control Plane API (Mixer) used for policy enforcement and telemetry collection
- Certificate provisioning and service-to-service encryption using mutual TLS.
- A batteries included monitoring experience using prometheus and grafana.

Reply:
Does mixer also implement the Envoy APIs?   I think not...